mlir/test/Target/LLVMIR/openmp-private.mlir

```diff
@@ -0,0 +1,21 @@
+// Test code-gen for `omp.parallel` ops with delayed privatizers (i.e. using
+// `omp.private` ops).
+
+// RUN: mlir-translate -mlir-to-llvmir -split-input-file %s | FileCheck %s
```

Comment:
We have a FileCheck here. We should probably add the IR against which the checks should happen. It would help in understanding the IR generation as well.

Reply:
Maybe I did not understand what you mean, but there are IR checks for generated code below in the file.

Reply:
It is okay now. Somehow, in the version I reviewed and cloned locally, I did not have any `CHECK` as such. Hence the comment. 

Closing it now...